### PR TITLE
Importing copyright tooling from datadog-agent repo

### PR DESCRIPTION
**Pull request by hush-hush**
Importing copyright tooling from datadog-agent repo

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
-// under the Apache License Version 2.0.
+// under the BSD 3-Clause License.
```
Comment (pgimalac) [inline main.go:2]: Not sure this change should be in this PR, but the license of the repo is really BSD3 so the fix seems correct

File: tasks/copyright.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright {datetime.datetime.now().year}-present Datadog, Inc.
+// Copyright (c) 2021, RapDev.IO
+""".strip()
+
+COPYRIGHT_REGEX = [
+    r'^// Unless explicitly stated otherwise all files in this repository are licensed$',
+    r'^// under the BSD 3-Clause License\.$',
+    r'^// This product includes software developed at Datadog \(https://www\.[Dd]atadoghq\.com/\)\.$',
+    r'^// Copyright 20[1-3][0-9]-([Pp]resent|20[1-3][0-9]) Datadog, (Inc|Inmetrics)\.$',
```
Comment (pgimalac) [inline tasks/copyright.py:23]: I think that's not wanted, even in the agent repo every copyright is `Datadog, Inc`
```suggestion
    r'^// Copyright 20[1-3][0-9]-([Pp]resent|20[1-3][0-9]) Datadog, Inc\.$',
```

The only matches I found are in the archived repo of the security agent 😄 
https://github.com/DataDog/datadog-security-agent/blob/f9f4adcbdec854cfa9c5981a40d2f4871ef04e7b/pkg/util/containers/providers/windows/dummy.go#L4

**Review by pgimalac — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (hush-hush) [reply main.go:2]: Testing the new linter it found this issue.

**Pull request merged**